rest-api-spec/pom.xml

```diff
@@ -0,0 +1,37 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.elasticsearch</groupId>
+  <artifactId>rest-api-spec</artifactId>
+  <version>2.0.0-SNAPSHOT</version>
+  <name>Elasticsearch Rest API Spec</name>
+  <parent>
+      <groupId>org.sonatype.oss</groupId>
+      <artifactId>oss-parent</artifactId>
```

Comment:
I tried that but the build then fails, I think because there is a circular dependency as rest-api-spec depends on elasticsearch-parent and elasticsearch-parent references rest-api-spec in the maven-remote-resources-plugin.